src/tactic/squeeze.lean

```diff
@@ -21,19 +21,25 @@ meta def loc.to_string : loc → string
 namespace tactic
 namespace interactive
 
-meta def erase_simp_arg (s : name_set) : simp_arg_type → name_set
-| simp_arg_type.all_hyps := s
-| (simp_arg_type.except a) := s
-| (simp_arg_type.expr e) :=
-match e.get_app_fn e with
-| (expr.const n _) := s.erase n
-| _ := s
-end
+/-- Erase names in the set which occur as a constant in the arguments. -/
```

Comment:
Sorry, not sure I understand this. What does it mean to "occur as a constant in the arguments"? What arguments? I think the comment would be better phrased as
```
/-- `erase_simp_args hs s` removes from `s` each name `n` such that ...
```
where `...` explains "`n` occurs as a constant in the arguments."